lib/src/mobile_scanner_controller.dart

```diff
@@ -596,8 +708,9 @@ class MobileScannerController extends ValueNotifier<MobileScannerState> {
   /// device. This can be used to determine which lens types can be used
   /// with the scanner.
   ///
-  /// Returns an empty set if the device has no cameras, or if the platform
-  /// does not support querying available lens types.
+  /// Returns an empty set if the device has no cameras. On platforms
+  /// that do not support querying specific lens types, returns a set
+  /// containing only `CameraLensType.any` if cameras are available.
```

Comment:
This can become a clickable doc link.

```suggestion
  /// containing only [CameraLensType.any] if cameras are available.
```

Reply:
Solved